MANUAL.txt

```diff
@@ -679,7 +679,7 @@ header when requesting a document from a URL:
     necessary, and adjust the images references in the document
     so they point to the extracted files.  Media are downloaded,
     read from the file system, or extracted from a binary
-    container (e.g. docx), as needed.  The original file paths
+    container (e.g. docx, ipynb), as needed.  The original file paths
```

Comment:
Strictly speaking, ipynb is not a "binary container".  Maybe remove the word "binary" on 681?